client/src/components/Draftail/index.js

```diff
@@ -113,6 +114,23 @@ const initEditor = (selector, originalOptions, currentScript) => {
       return { ...plugin, ...type };
     });
 
+    // Only initialise the character count / max length on fields explicitly requiring it.
+    if (field.hasAttribute('maxlength')) {
```

Comment:
For most other editor configuration we we use the widget’s `options` field. I thought `maxlength` would be ok to use straight from the DOM since it’s something Django adds even to hidden input fields.